exporter/jaeger/jaeger.go

```diff
@@ -37,8 +37,13 @@ const defaultServiceName = "OpenCensus"
 type Options struct {
 	// Endpoint is the Jaeger HTTP Thrift endpoint.
 	// For example, http://localhost:14268.
+	// Deprecated: Use CollectorEndpoint instead.
```

Comment:
Add a new line before the deprecation notice to make it stand out on godoc.

```
// Endpoint is the Jaeger HTTP Thrift endpoint.
// For example, http://localhost:14268.		// For example, http://localhost:14268.
//
// Deprecated: Use CollectorEndpoint instead.
```

Reply:
Added the new line